### PR TITLE
Remove all uses of zoottelkeeper plugin

### PR DESCRIPTION
**Pull request by claremacrae**
### Changes made

Remove all sets of lines:

```
## MOC

%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
...
%% Zoottelkeeper: End of the autogenerated index file list  %%
```

And also sets where the was no preceding `## MOC`

```
%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
...
%% Zoottelkeeper: End of the autogenerated index file list  %%
```

### Motivation

For the background to this, see the discussion starting at [Pardon my having missed it, but why are we using zootelkeeper for this](https://discord.com/channels/686053708261228577/891399239198982195/904785982518345729).

We concluded that as no-one was using the whole graph in the published hub, the links generated by running zoottelkeeper were not worth the high time cost of configuring and running it.

## Checklist

- [x] I have renamed all attached images with descriptive file names (or I did not include any images)
- [x] Before creating a new note, I searched the vault (or I only edited existing notes)

### CODE REVIEW

**Comment by claremacrae** [comment]
See also #65 and #66 for follow-on steps...

File: 01 - Community/Obsidian Roundup/🗂️ Obsidian Roundup.md [hunk 1]
```diff
@@ -11,36 +11,3 @@ publish: true
 The Obsidian Roundup is a weekly newsletter written by written by moderator [[eleanorkonik|Eleanor Konik]]. It is intended to help Obsidian users stay in the loop about community events, code updates and workflow tips. Each issue represents a summary of potentially relevant information the community engaged with in a particular week. You can [subscribe](https://www.obsidianroundup.org/membership/) via email or RSS. 
 
 This archive lags slightly behind the one on the [Roundup's website](https://obsidianroundup.org/) but is interlinked with the rest of the Hub thanks to community efforts. You can contribute to that effort [here](https://github.com/obsidian-community/obsidian-hub/contribute).
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
--  [[01 - Community/Obsidian Roundup/2021-10-02  Premade Concept Hierarchies & a Virtual Community Meeting Space|2021-10-02  Premade Concept Hierarchies & a Virtual Community Meeting Space]]
```
Comment (argenos) [inline 01 - Community/Obsidian Roundup/🗂️ Obsidian Roundup.md:18]: Might make sense to keep this as a manual MOC, not sure. @eleanorkonik thoughts?

File: 01 - Community/Video Channels/🗂️ Video Channels.md [hunk 2]
```diff
@@ -9,12 +9,3 @@ publish: true
 # 🗂️ Video Channels
 
 #placeholder/description 
-
```
Comment (argenos) [inline 01 - Community/Video Channels/🗂️ Video Channels.md:12]: I would delete this file, the manual MOC can then be the one for YouTube channels

File: 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/🗂️ CSS Snippets.md [hunk 3]
```diff
@@ -9,16 +9,3 @@ publish: true
 # 🗂️ CSS Snippets
 
 #placeholder/description 
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
```
Comment (argenos) [inline 02 - Community Expansions/02.05 All Community Expansions/CSS Snippets/🗂️ CSS Snippets.md:15]: I'd keep these as a manual MOC for now (once more snippets are added, we'll have to adjust).

File: 04 - Guides, Workflows, & Courses/Courses/🗂️ Courses.md [hunk 4]
```diff
@@ -12,7 +12,3 @@ publish: true
 
 #placeholder/description 
 
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
```
Comment (argenos) [inline 04 - Guides, Workflows, & Courses/Courses/🗂️ Courses.md:15]: I'd keep this one as a manual MOC

File: 04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md [hunk 5]
```diff
@@ -9,30 +9,6 @@ tags:
 
 #placeholder/description 
 
-## MOC
-
```
Comment (argenos) [inline 04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md:13]: I'd also keep this one as a manual MOC to be refactored (probably needs headings to split the guides into useful things).

File: 🗂️ hub.md [hunk 6]
```diff
@@ -9,20 +9,3 @@ publish: true
 # 🗂️ hub
 
 #placeholder/description 
-
```
Comment (argenos) [inline 🗂️ hub.md:12]: I would delete this note.

File: 02 - Community Expansions/02.05 All Community Expansions/🗂️ 02.05 All Community Expansions.md [hunk 7]
```diff
@@ -11,12 +11,3 @@ publish: true
 This is a comprehensive directory of **all** expansions for Obsidian made by the Community – [[🗂️ Plugins|Plugins]], [[🗂️ Themes|Themes]], [[🗂️ CSS Snippets| CSS Snippets]], and [[🗂️ Auxiliary Tools|Auxilliary Tools]]. 
 
 Unsurprisingly, these are quite a lot! If you feel overwhelmed, you might want to browse the [[🗂️ 02 - Community Expansions|Community Expansions by category]] instead.
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
```
Comment (argenos) [inline 02 - Community Expansions/02.05 All Community Expansions/🗂️ 02.05 All Community Expansions.md:17]: Keeping this as a manual MOC?

File: 03 - Showcases & Templates/Publish Sites/🗂️ Publish Sites.md [hunk 8]
```diff
@@ -9,9 +9,3 @@ publish: true
 # 🗂️ Publish Sites
 
 #placeholder/description 
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
--  [[03 - Showcases & Templates/Publish Sites/Obsidian Garden|Obsidian Garden]]
```
Comment (argenos) [inline 03 - Showcases & Templates/Publish Sites/🗂️ Publish Sites.md:16]: I'd keep this as a manual MOC

File: 00 - Contribute to the Obsidian Hub/01 Templates/🗂️ 01 Templates.md [hunk 9]
```diff
@@ -6,29 +6,3 @@ tags:
 ---
 
 # 🗂️ 01 Templates
-
```
[the remark this answers is not in the document]
Comment (claremacrae) [reply 00 - Contribute to the Obsidian Hub/01 Templates/🗂️ 01 Templates.md:9]: There are backlinks to it, which I though were useful...

image

As far as I understand it, these 🗂️ files provide the ability for other .md files to link to a folder, and that is done in multiple locations throughout this vault.

So I had wondered whether a later step might add some useful content to this, explaining its purpose...

File: 00 - Contribute to the Obsidian Hub/02 Attachments/🗂️ 02 Attachments.md [hunk 10]
```diff
@@ -9,56 +9,3 @@ publish: true
 # 🗂️ 02 Attachments
 
 #placeholder/description 
-
```
[the remark this answers is not in the document]
Comment (claremacrae) [reply 00 - Contribute to the Obsidian Hub/02 Attachments/🗂️ 02 Attachments.md:12]: OK. What would you do with the links to it?

image

File: 01 - Community/Events/🗂️ Events.md [hunk 11]
```diff
@@ -8,11 +8,3 @@ tags:
 # 🗂️ Events
 
 Overview of the various Events happening in the Obsidian Community.
-
-## MOC
-
-%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
```
[the remark this answers is not in the document]
Comment (claremacrae) [reply 01 - Community/Events/🗂️ Events.md:14]: Done: https://github.com/obsidian-community/obsidian-hub/commit/53ee620115e344393704d5c0d7217f8566cb36f2

File: 01 - Community/Video Channels/🗂️ Video Channels.md (hunk 2 above)
Comment (claremacrae) [reply 01 - Community/Video Channels/🗂️ Video Channels.md:12]: OK. Links to this: 
image

File: 04 - Guides, Workflows, & Courses/Courses/🗂️ Courses.md (hunk 4 above)
Comment (claremacrae) [reply 04 - Guides, Workflows, & Courses/Courses/🗂️ Courses.md:15]: Thanks.

Done: https://github.com/obsidian-community/obsidian-hub/commit/2ea8a007691a5c204e48c2af057f933c6f636815

File: 04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md (hunk 5 above)
Comment (claremacrae) [reply 04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md:13]: Done: https://github.com/obsidian-community/obsidian-hub/commit/2b359404646307d3de317f7a9ae1f60007790b13

File: 🗂️ hub.md (hunk 6 above)
Comment (claremacrae) [reply 🗂️ hub.md:12]: Done - e469086461ac99f428aac65ace9ee2e195582be4

**Comment by claremacrae** [comment]
Closing this PR, after discussion in Discord.

Basically, limitations in the UI of the file/folder listing on mobile means that we still need some kind of automated file listing in each folder.

As separate steps, I will:

1. Set up a different PR that applies a few of the things I learned this morning - like removing the empty Zoot blocks in non-MOC files...
2. Work on a proof-of-concept Python script that generates the text we want, and can be run from a console or even GitHub Actions

**Comment by claremacrae** [comment]
@chrisgrieser Please ignore the review request here - this PR has been closed...

**Pull request closed without merging**